components/script/dom/extendablemessageevent.rs

```diff
@@ -123,6 +123,21 @@ impl ExtendableMessageEvent {
         );
         Extendablemessageevent.upcast::<Event>().fire(target);
     }
+
+    pub fn dispatch_error(target: &EventTarget, scope: &GlobalScope, message: HandleValue) {
+        let ExtendableMsgEvent = ExtendableMessageEvent::new(
```

Comment:
We can do something similar to what is currently done in `messageevent.rs`, by:

1. In `ExtendableMessageEvent.webidl`, un-comment the `ports` attribute of `ExtendableMessageEventInit`.
2. Here, use `let init = ExtendableMessageEvent::MessageEventInit::empty();`
3. then use the data from init when calling `ExtendableMessageEvent::new`.

Reply:
Okay, I'll rewrite and cleanup the newlines. My bad. Thank you!